### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio on "Messages" page

### ISSUE THREAD

**Issue by elouiestudent** [CMU-313/Mayan-EDMS#32]
The current Lighthouse accessibility score is 81. The Lighthouse-reported problem being targeted is that the "Actions" 
 dropdown button's background color does not have a sufficient contrast ratio with the text color of the button.
